etc/ci/upload_nightly.sh

```diff
@@ -13,6 +13,7 @@ usage() {
 
 upload() {
     s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    s3cmd put "${2}" "s3://servo-builds/nightly/${1}/servo-latest${3}"
```

Comment:
nit: Do not include the leading period in the various "${extension}" variables but put it in here,
i.e. `servo-latest.${3}`